package.json

```diff
@@ -33,6 +34,8 @@
     "path": "^0.12.7"
   },
   "devDependencies": {
+    "@aws-sdk/client-dynamodb": "^3.202.0",
+    "@aws-sdk/util-dynamodb": "^3.204.0",
```

Comment:
why those deps?

Reply:
We don't need them, I forgot to remove it.